common/define-grammar.js

```diff
@@ -114,6 +114,9 @@ module.exports = function defineGrammar(dialect) {
         field('arguments', optional($.arguments))
       )),
 
+      _augmented_assignment_lhs: ($, previous) => choice(...previous.members.concat([$.non_null_expression])),
+      _lhs_expression: ($, previous) => choice(...previous.members.concat([$.non_null_expression])),
```

Comment:
I think both of these could be written `choice(previous, $.non_null_expression)`. The nesting of choices doesn't matter, so there's no need to 'flatten' it out.

Reply:
Suggestion taken